src/Compilers/CSharp/Test/Syntax/Parsing/FileModifierParsingTests.cs

```diff
@@ -2377,7 +2428,6 @@ void M()
         EOF();
     }
 
-    // PROTOTYPE(ft): confirm whether this breaking change is acceptable
```

Comment:
>PROTOTYPE(ft): confirm whether this breaking change is acceptable [](http://example.com/codeflow?start=7&length=65)

Was this a stale comment? It looks like we report these binding errors today.

Reply:
I think whether 'file' in a type position parses as a type depends on context, and for simplicity we disallow you from using it as a type name. It ends up continuing to parse as a type in this scenario. I'll update the breaking change doc.